src/layouts/Layout.astro

```diff
@@ -56,10 +56,20 @@ const { title, navless } = Astro.props;
     ) }
     <slot></slot>
     <footer class="container">
-      <span><a class="sneaky" href="/join">Join</a></span>
-      <span><a class="sneaky" href="/about">About</a></span>
-      <span><a class="sneaky" href="/members">Members</a></span>
-      <span><a class="sneaky" href="/endorsements">Endorsements</a></span>
+      <div class="footer-nav">
+        <span><a class="sneaky" href="/join">Join</a></span>
+        <span><a class="sneaky" href="/about">About</a></span>
+        <span><a class="sneaky" href="/members">Members</a></span>
+        <span><a class="sneaky" href="/endorsements">Endorsements</a></span>
+      </div>
+      <div class="footer-socials">
```

Comment:
Can we get a little more vertical padding between nav and this and this and bottom?

<img width="821" alt="image" src="https://github.com/user-attachments/assets/99735997-7f14-4f73-9ab4-5f6ba3edf617">